src/Symfony/Component/Console/Helper/Dumper.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Helper;
+
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\VarDumper\Cloner\VarCloner;
+use Symfony\Component\VarDumper\Dumper\CliDumper;
+
+/**
+ * @author Roland Franssen <franssen.roland@gmail.com>
+ */
+final class Dumper
+{
+    private $output;
+    private $handler;
+
+    public function __construct(OutputInterface $output, callable $handler = null)
+    {
+        $this->output = $output;
+        $this->handler = $handler ?? function ($variable) {
+            if (class_exists(CliDumper::class)) {
+                $cloner = new VarCloner();
+                $dumper = new CliDumper(null, null, CliDumper::DUMP_LIGHT_ARRAY | CliDumper::DUMP_COMMA_SEPARATOR);
+                $dumper->setColors($this->output->isDecorated());
+
+                return rtrim($dumper->dump($cloner->cloneVar($variable)->withRefHandles(false), true));
+            }
+
+            if (null === $variable) {
+                return 'null';
+            } elseif (true === $variable) {
```

Comment:
`elseif` with `return` is point less.
```suggestion

            if (null === $variable) {
                return 'null';
            }

            if (true === $variable) {
                return 'true';
```

Reply:
not pointless in terms of no. of lines :) i believe this is a CS issue and for now it's 1 - 1 :)

Reply:
```php
switch (true) {
   case class_exists(CliDumper::class):
       $cloner = new VarCloner();
       $dumper = new CliDumper(null, null, CliDumper::DUMP_LIGHT_ARRAY | CliDumper::DUMP_COMMA_SEPARATOR);
       $dumper->setColors($this->output->isDecorated());

       return rtrim($dumper->dump($cloner->cloneVar($variable)->withRefHandles(false), true));
   case null === $variable:
       return 'null'
   case true === $variable:
       return 'true'
   case false === $variable:
       return 'false'
   case \is_string($variable):
       return '"'.$variable.'"';
   default:
       return rtrim(print_r($variable, true));
}
```

? 😄

Reply:
Also you should probably instantiate `$cloner` and `$dumper` only once, out of the handler and rather create a dedicated var dumper handler if `class_exists(CliDumper::class)` is true rather than having this condition executed each time the handler is called.